python/tvm/relay/frontend/onnx.py

```diff
@@ -449,6 +449,8 @@ class Upsample(OnnxOpConverter):
     @classmethod
     def _impl_v7(cls, inputs, attr, params):
         scales = attr.get('scales')
+        if not scales:
```

Comment:
inputs[1] may be mostly available in params.

Can you attempt taking it from params and proceeding to make it compatible for opset-8 ?

Reply:
But how to get the scale value from input[1],  and how to create the new input? Is following code snippet resonable? Thanks.
```python
    def _impl_v7(cls, inputs, attr, params):
        scales = attr.get('scales')
        if not scales:
            #Here we are going to higher OPSET version.
            assert len(inputs) == 2, "Upsample op take 2 inputs, {} given".format(len(inputs))
            scales = params[inputs[1].name_hint].asnumpy()
            inputs = inputs[:1]
```

Reply:
@gemfield yes, above code look good. Add it and let the CI pass for it.
Also please try adding a test case too for it.